devportal/devportal-domain-service/ballerina/config.bal

```diff
@@ -49,4 +52,23 @@ type DatasourceConfiguration record {
 type APKConfiguration record {
     ThrottlingConfiguration throttlingConfiguration;
     DatasourceConfiguration datasourceConfiguration;
+    TokenIssuerConfiguration tokenIssuerConfiguration;
+    KeyStores keyStores;
 };
+
+public type TokenIssuerConfiguration record {|
+    string issuer = "https://apim.wso2.com/oauth2/token";
+    string audience = "https://apim.wso2.com/oauth2/token";
+    string keyId = "gateway_certificate_alias";
+    decimal expTime = 3600;
```

Comment:
isn't this should be configurable

Reply:
yes, this provided through configurations these are default values